app/views/idv/cac/choose_method.html.erb

```diff
@@ -1,24 +1,45 @@
 <%= render 'shared/maintenance_window_alert' do %>
+  <style>
+    .flex-container {
+        display: flex;
+    }
+    @media (max-width: 30em) {
+        .flex-container {
+            flex-direction: column;
+        }
+    }
+  </style>
   <h1 class="h3 mt0 mb1"><%= t('cac_proofing.headings.choose_method') %></h1>
-  <%= t('cac_proofing.info.choose_method') %>
+  <p><%= t('cac_proofing.info.choose_method') %></p>
+  <p><%= t('cac_proofing.info.disclaimer') %></p>
+  <div class="flex-container mt3 mb3">
```

Comment:
I think rather than creating our own flex container, we can use the [USWDS layout grid 12-column grid](https://designsystem.digital.gov/utilities/layout-grid/) for a 1/3 , 2/3 layout:

```html
<div class="grid-row">
  <div class="grid-col-4"><img></div>
  <div class="grid-col-8">...</div>
</div>
```

It might also help avoid some content resizing I'm seeing on first load.

Reply:
I tried to use this system but I couldn't figure out how to make it work as well as the simple flexbox I built. I'm leaving it for now. If you want to take a look at making it work with the grid system you can but I couldn't duplicate what I had already done here quickly.

Reply:
I pushed some revisions in c1da0bf. This should also solve the issue with style CSP.